lmp/jobserv.yml

```diff
@@ -342,25 +304,12 @@ triggers:
               - beaglebone-yocto
               - generic-arm64
               - intel-corei7-64
-              - imx6ullevk
```

Comment:
This was still need for PR's targeting the EOL (kirkstone) but I nevertheless agree that it should be removed because without these machines in the most recent branches the PRs destined for them will fail.

Relates to https://github.com/foundriesio/ci-scripts/issues/399

Reply:
Remove please.